src/screens/HomeScreen/component.js

```diff
@@ -0,0 +1,145 @@
+// @flow
+import React, { PureComponent } from "react";
+import {
+  StyleSheet,
+  SafeAreaView,
+  ScrollView,
+  View,
+  TouchableOpacity
+} from "react-native";
+import type { NavigationScreenProp, NavigationState } from "react-navigation";
+import Text from "../../components/Text";
+import type { Event, Asset } from "../../data/event";
+import EventTile from "../../components/EventTile";
+import { cardBgColor, imageBgColor } from "../../constants/colors";
+import { EVENT_LIST, EVENT_DETAILS } from "../../constants/routes";
+
+const locale = "en-GB";
+
+type Props = {
+  navigation: NavigationScreenProp<NavigationState>,
+  events: Event[],
+  loading: boolean,
+  getAssetById: string => Asset
+};
+
+type ColumnProps = {
+  column: number,
+  events: Event[],
+  onPress: (eventId: string) => void,
+  getAssetById: string => Asset
+};
+const Column = ({ column, events, onPress, getAssetById }: ColumnProps) => (
+  <View style={column === 0 ? styles.viewLeft : styles.viewRight}>
+    {events.map(event => (
+      <TouchableOpacity
+        key={event.sys.id}
+        style={styles.tileWrapper}
+        onPress={() => onPress(event.sys.id)}
```

Comment:
Sharing some things we learned from the event list:

In a scrolling view it is good to add `delayPressIn={50}` to the `TouchableOpacity`. Otherwise you will see the opacity effect, even when you are just scrolling. This is a bit distracting and does not really feel native. The delay helps.

I think we should add accessibility attributes to all custom buttons. Otherwise they are treated as text by screen readers. We need `accessibilityTraits` for iOS and `accessibilityComponentType` for Android. You can just copy the attributes from the `EventList` component:

https://github.com/redbadger/pride-london-app/blob/1e56cf838713589fcceb0abdf02ce85553f2e069/src/components/EventList.js#L33-L35